ARM.CMSIS.pdsc

```diff
@@ -3870,11 +3870,11 @@ and 8-bit Java bytecodes in Jazelle state.
       </attributes>
     </example>
 
-    <example name="NN Library GRU" doc="readme_iar.txt" folder="CMSIS/NN/Examples/IAR/iar_nn_examples/gru">
+    <example name="gru" doc="readme_iar.txt" folder="CMSIS/NN/Examples/IAR/iar_nn_examples/gru">
```

Comment:
I think the example name should be slightly more specific and expressive.  The name is used to identify the example project in pack explorer.